### PR TITLE
Setting improvements

### PR DESCRIPTION
**Pull request by mnoack**
Ontop of https://github.com/sealink/active_setting/pull/3

### CODE REVIEW

**Comment by alvinypyim** [comment]
👍

**Review by alvinypyim — APPROVED** [review]

File: spec/setting_spec.rb [hunk 1]
```diff
@@ -62,12 +62,17 @@
...
     context 'when array of integers' do
       let(:subtype) { :integer }
       let(:raw_value) { '1,2,3' }
       it { is_expected.to eq [1, 2, 3] }
+
+      context 'when no value, so default' do
+        let(:raw_value) { nil }
+        it { is_expected.to eq [] }
+      end
     end
```
Comment (kieronhowland) [inline spec/setting_spec.rb:87]: worth adding a test where the default is a non-empty Array

Comment (mnoack) [reply spec/setting_spec.rb:87]: Good point. Test added :)

**Review by kieronhowland — APPROVED** [review]

**Comment by alxberardi** [comment]
👍 once @kieronhowland 's comment is addressed

**Review by alxberardi — CHANGES_REQUESTED** [review]
Please address @kieronhowland 's comment about the missing spec

**Review by alxberardi — APPROVED** [review]
